cinema/views.py

```diff
@@ -43,6 +57,25 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if genres:
+            genres = _params_to_ints(genres)
```

Comment:
In `MovieViewSet`, before calling `_params_to_ints(genres)`, ensure that `genres` is not `None` to prevent potential errors.